common/src/main/java/com/datatorrent/common/util/FSStorageAgent.java

```diff
@@ -153,6 +154,16 @@ public void delete(int operatorId, long windowId) throws IOException
   public long[] getWindowIds(int operatorId) throws IOException
   {
     Path lPath = new Path(path + Path.SEPARATOR + String.valueOf(operatorId));
+    try {
```

Comment:
during application restart as the new directory for current application is not created, the getWindowIds will throw an exception. hence this try is added to catch FileNotFoundException